### DISCUSSION TITLE
Linus Tech Tips

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#28] [Q&A]
Commentaires sur ce quiz. 

**Accès**
- Serveur de développement (protégé) : https://timunix3.csfoy.ca/~quiz-TIM/Z0ul0u25/index.html 
- Dépôt Github (privé pour le moment) : https://github.com/Z0ul0u25/quiz-tim  
- Github.io ?

**Comment by kateundercoffler** [comment]
Salut :)  Le quiz est bien fait.  Quand j'utilise le touche clavier pour faire le quiz, ça semble correct.  De plus, le contenu est intéressant.  Pour améliorer :  Comme je t'ai mentionné dans la classe, ton police de caractère pour les paragraphe ou main partie du quiz est trop condensé et un peu difficile à lire.  Au lieu de mettre un interlettrage, c'est mieux de choisir un autre police pour aider la lisibilité. https://tympanus.net/codrops/2012/03/20/condensed-fonts-the-good-the-bad-the-ugly/

Bien fait Philippe ! 👍

**Comment by evefevrier** [comment]
Une petite observation:
- au clic d'un choix de réponse les autres choix de réponses ne devraient pas bouger... Facile à éviter en prévoyant un border d'un même épaisseur de couleur _transparent_.

**Comment by Z0ul0u25** [comment]
>     * Dépôt Github (privé pour le moment) : https://github.com/Z0ul0u25/quiz-tim
> 
>     * Github.io ?

Publié : [z0ul0u25.github.io/quiz-tim](https://z0ul0u25.github.io/quiz-tim/Z0ul0u25/index.html)
